pkg/operator/connectivitycheckcontroller/connectivity_check_controller.go

```diff
@@ -208,6 +172,26 @@ func (c *connectivityCheckController) Sync(ctx context.Context, syncContext fact
 	return nil
 }
 
+func ensureConnectivityCheckCRDExists(ctx context.Context, syncContext factory.SyncContext, client *apiextensionsclient.Clientset) error {
+	_, err := client.ApiextensionsV1().CustomResourceDefinitions().Get(ctx, podnetworkconnectivitychecksCRDName, metav1.GetOptions{})
+	if err != nil && !errors.IsNotFound(err) {
+		return err
+	}
+	if errors.IsNotFound(err) {
+		// create the podnetworkconnectivitycheck crd that should exist
+		applyResults := resourceapply.ApplyDirectly(
+			resourceapply.NewClientHolder().WithAPIExtensionsClient(client),
+			syncContext.Recorder(),
+			func(name string) ([]byte, error) { return bindata.Asset(name) },
+			"pkg/operator/connectivitycheckcontroller/manifests/controlplane.operator.openshift.io_podnetworkconnectivitychecks.yaml",
```

Comment:
This needs that the version in the manifests directory under pkg/operator/... matches the vendored version in the CNO repo. How will we ensure that they stay in sync?

Reply:
> This needs that the version in the manifests directory under `pkg/operator/...` matches the vendored version in the CNO repo. How will we ensure that they stay in sync?

The make `verify` target ensures that.

Reply:
:+1